scanner/Makefile

```diff
@@ -41,6 +41,12 @@ endif
 # The namespace to use for targets that require it (E2E, certs, etc).
 SCANNER_NAMESPACE ?= stackrox
 
+# The init bundle version to use when building the db image.
+SCANNER_DB_INIT_BUNDLE_VERSION ?= dev
```

Comment:
This will likely disappear and be replaced by a Makefile target like `make image-db-vulns`.
The target will download the correct vulnerability bundle based on the current tag in the repo.
The workflow will use it to download the vulns, and pass it along to the updater.